engine/src/main/scala/cromwell/engine/io/gcs/GcsBatchFlow.scala

```diff
@@ -81,16 +83,22 @@ class GcsBatchFlow(batchSize: Int, scheduler: Scheduler, onRetry: IoCommandConte
     * object when the internal queue is "dirty" with a .size() > 0.
     */
   private def newBatchRequest(): BatchRequest = {
-    val storage = new Storage.Builder(
+    val builder = new Storage.Builder(
       GcsStorage.HttpTransport,
       JacksonFactory.getDefaultInstance,
       httpRequestInitializer
     ).setApplicationName(applicationName)
-
-    storage.build().batch()
+    val client = builder.build()
+    client.batch(builder.getHttpRequestInitializer)
```

Comment:
Do we know what the defaults are if we don't pass in an `httpRequestInitializer`? I tried to research this and came up with nothing useful.

Reply:
20 seconds: https://github.com/googleapis/google-http-java-client/blob/v1.38.0/google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java#L151

That's the read timeout. Connect timeout is right above it. Also referenced that link in the google ticket I filed back when we didn't know what was going on: https://console.cloud.google.com/support/cases/detail/27230142?project=broad-dsde-cromwell-dev

Reply:
Sheesh. Is this really only discoverable by reading the source? Thanks for finding the answer.

Since the timeout is in config and therefore deployment-specific, it's hard to know how this fix will affect existing deployments, but it's hard to imagine any negative impact.